### PR TITLE
make activity stream action_node deterministic for Swagger docs

### PR DESCRIPTION
**Pull request by ryanpetrello**

### CODE REVIEW

File: awx/main/tests/docs/test_swagger_generation.py [hunk 1]
```diff
@@ -176,4 +176,8 @@ def teardown_class(cls):
                 r'\1xxxx\3',
                 data
             )
+            data = re.sub(
+                r'"action_node": "awx-[^"]+"',
```
Comment (ryanpetrello) [inline awx/main/tests/docs/test_swagger_generation.py:180]: zuul generates randomized hostnames for the containers, so this was making example activity stream output non-deterministic

**Comment by ryanpetrello** [comment]
@AlanCoding this is fairly chicken-and-egg; `awx-detect-schema-change` won't actually pass until *after* we upload a new reference schema with `"action_node": "awx".

**Comment by AlanCoding** [comment]
> awx-detect-schema-change won't actually pass until after we upload a new reference schema with `"action_node": "awx".

Oh yeah, totally expected. I did look to confirm that no places were missed.

```
--- reference-schema.json	2019-06-06 14:46:36.925158361 +0000
+++ schema.json	2019-06-06 14:46:35.434051551 +0000
@@ -162,7 +162,7 @@
                 "previous": null,
                 "results": [
                   {
-                    "action_node": "awx-push-new-schema-4013",
+                    "action_node": "awx",
                     "changes": {
                       "custom_virtualenv": null,
                       "description": "test-org-desc",
@@ -195,7 +195,7 @@
                     "url": "/api/v2/activity_stream/1/"
                   },
                   {
-                    "action_node": "awx-push-new-schema-4013",
+                    "action_node": "awx",
                     "changes": {
                       "email": "",
                       "first_name": "",
@@ -269,7 +269,7 @@
             "description": "",
             "examples": {
               "application/json": {
-                "action_node": "awx-push-new-schema-4013",
+                "action_node": "awx",
                 "changes": {
                   "action": "associate",
                   "object1": "user",
```

and it looks good!

**Review by AlanCoding — APPROVED** [review]

**Comment by ryanpetrello** [comment]
regate

**Pull request merged**